src/lsp/cobol_cfg/cfg_options.ml

```diff
@@ -0,0 +1,23 @@
+(******************************************************************************)
+(*                                                                            *)
+(*     Copyright (c) 2021-2023 OCamlPro SAS                                   *)
+(*                                                                            *)
+(*     All rights reserved.                                                   *)
+(*     This file is distributed under the terms of the                        *)
+(*     OCAMLPRO-NON-COMMERCIAL license.                                       *)
+(*                                                                            *)
+(******************************************************************************)
+
+type transformation =
+  | Descendents of int
+  | Neighborhood of int
+
+type t = {
+  hide_unreachable: bool;
+  collapse_fallthru: bool;
+  shatter_hubs: int option;
```

Comment:
`shatter_hubs` is a kind of threshold on a measure on produced graphs. So its name should be correspond do that. Is this something like `node_degree_upper_limit` (counting every arc entering/leaving a node)?